core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.spec.ts

```diff
@@ -148,21 +148,17 @@ describe('Translation Suggestion Review Modal Controller', function() {
         .toHaveBeenCalledWith('Translation');
     });
 
-    it('should notify user on failed suggestion activities', function() {
+    it('should notify user on failed suggestion update', function() {
       const error = {
         data: {
           error: 'Error'
         }
       };
-      spyOn(AlertsService, 'clearWarnings');
-      spyOn(AlertsService, 'addWarning');
 
       $scope.showTranslationSuggestionUpdateError(error);
 
-      expect(AlertsService.clearWarnings).toHaveBeenCalled();
-      expect(
-        AlertsService.addWarning).toHaveBeenCalledWith(
-        'Invalid Suggestion: Error');
+      expect($scope.errorFound).toBeTrue();
+      expect($scope.errorMessage).toBe('Invalid Suggestion: Error');
```

Comment:
Please add a pre-check for these variables, i.e., a check before `$scope.showTranslationSuggestionUpdateError` is executed

Reply:
Done